lib/LedgerSMB/Report/GL.pm

```diff
@@ -333,7 +333,7 @@ sub run_report{
     my @rows = $self->call_dbmethod(funcname => 'report__gl');
     for my $ref(@rows){
         if ($ref->{amount} < 0){
-            $ref->{debits} = $ref->{amount} * -1;
+            $ref->{debits} =  -$ref->{amount};
```

Comment:
Sorry my question about what the `-$ref` is negating was somewhat rhetorical.
It may not be a problem in Perl, but in many languages, the exact behavior of that construct depends on many things, and the general recommendation is that brackets be used to ensure the correct scoping.